.vscode/extensions.json

```diff
@@ -2,6 +2,7 @@
   "recommendations": [
     "esbenp.prettier-vscode",
     "dbaeumer.vscode-eslint",
-    "DavidAnson.vscode-markdownlint"
+    "DavidAnson.vscode-markdownlint",
+    "hbenl.vscode-mocha-test-adapter"
```

Comment:
Does this file support trailing commas, by the way?

Reply:
Not sure but I will keep exiting. I had some chat with @tomaciazek few days ago regarding running and debugging tests and he was already using the same extension. I found that after I had some bad experience with two others. @priyamsahoo also knows as we has a pair-programming session mid week to narrow down testing issue with ALS. 

I would count that as very useful tool. Also recommandation does not force user to install it. Try, it works best on ALS but I will do some follow-ups to make it work here. The issues are not from extension but the fact that only one test suite is using mocha "by the book", the other one is more problematic.

Reply:
BTW, see https://github.com/ansible/ansible-language-server/blob/main/.vscode/extensions.json#L6

Reply:
Done. Feel free to tune it if you have extra time. I am sure everyone from the team will understand it.

Reply:
Looks like you didn't do it after all.